src/scripts/domain/moveCalculator.ts

```
@@ -126,4 +126,199 @@ export class MoveCalculator {
+  }
+
+  private static addPawnAttackMoves(
+    board: BoardInfo,
+    moves: Move[],
+    piece: PieceInfo,
+  ) {
+    const toRemove: PieceInfo[] = [];
+    const pos = piece.pos;
+    const path: Vector2[] = [pos.clone()];
+
+    for (const direction of this.cardinalDirections) {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)